src/packages/client/src/generator.ts

```diff
@@ -25,10 +28,9 @@ generatorHandler({
     }
   },
   async onGenerate(options) {
-    if (debugEnabled) {
-      console.log('__dirname', __dirname)
-      console.log(eval(`__dirname`)) // tslint:disable-line
-    }
+    // TODO @timsuchanek Any idea what this is for?
+    debug('__dirname', __dirname)
+    debug(eval(`__dirname`)) // tslint:disable-line
```

Comment:
@timsuchanek

Reply:
I chatted to Tim, just need to remove this